exposestrategy/ingress.go

```diff
@@ -243,31 +214,28 @@ func (s *IngressStrategy) Add(svc *api.Service) error {
 
 	servicePort, err := strconv.Atoi(exposePort)
 	if err != nil {
-		return errors.Wrapf(err, "failed to convert the exposed port '%s' to int", exposePort)
+		return "", errors.Wrapf(err, "failed to convert the exposed port '%s' to int", exposePort)
```

Comment:
Why the "",?

Reply:
Looks like it's purely so that the function can also return the `fullHostName` at the end.  Appears that the test suite asserts against this.

```go
func (s *IngressStrategy) createIngress(cli client.IngressNamespacer, svc *api.Service) (string, error)
# down at the end
	return fullHostName, nil
```